README.md

```diff
@@ -630,12 +646,12 @@ Discharging efficiency of storage device at bus ![b](https://render.githubuserco
 All equations apply over all entries in the indexed sets unless otherwise listed.
 
 - ![0 \le g_{i,s,t} \le g_{i,s,t}^{\text{max}}](https://render.githubusercontent.com/render/math?math=0%5Cle%20g_%7Bi%2Cs%2Ct%7D%5Cle%20g%5E%7B%5Ctext%7Bmax%7D%7D_%7Bi%2Cs%2Ct%7D):
-generator segment power is non-negative and less than the segment width.
-- ![0 \le s_{b,t} \le a^{\text{shed}} \cdot d_{b,t}](https://render.githubusercontent.com/render/math?math=0%20%5Cle%20s_%7Bb%2Ct%7D%20%5Cle%20a%5E%7B%5Ctext%7Bshed%7D%7D%20%5Ccdot%20d_%7Bb%2Ct%7D):
-load shed is non-negative and less than the demand at that bus, if load shedding is enabled.
+Generator segment power is non-negative and less than the segment width.
+- ![0 \le s_{b,t} \le a^{\text{shed}} \cdot ( d_{b,t} + \delta_{b,t}^{\text{up}} - \delta_{b,t}^{\text{down}} )](https://render.githubusercontent.com/render/math?math=0%20%5Cle%20s_%7Bb%2Ct%7D%20%5Cle%20a%5E%7B%5Ctext%7Bshed%7D%7D%20%5Ccdot%20%28%20d_%7Bb%2Ct%7D%20%2B%20%5Cdelta_%7Bb%2Ct%7D%5E%7B%5Ctext%7Bup%7D%7D%20-%20%5Cdelta_%7Bb%2Ct%7D%5E%7B%5Ctext%7Bdown%7D%7D%20%29):
+Load shed is non-negative and less than the demand at that bus, if load shedding is enabled.
```

Comment:
`less than the demand at that bus` -> `less than the demand at that bus (including the impact of demand flexibility)`

Reply:
Done.